test/EFCore.PG.FunctionalTests/Migrations/MigrationsNpgsqlTest.cs

```diff
@@ -277,6 +277,68 @@ public virtual async Task Create_table_with_system_column()
 """);
     }
 
+    [Fact]
+    public virtual async Task Create_table_with_oid_column_when_pg_lower_than_12()
+    {
+        if (TestEnvironment.PostgresVersion.IsUnder(12))
```

Comment:
nit: if keeping both tests separate, invert this condition: simply return immediately if we're on 12 or above.